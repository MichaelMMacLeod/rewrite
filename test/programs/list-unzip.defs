(def (unzip (list (pair $x $y) ..))
  (pair (list $x ..)
        (list $y ..)))
