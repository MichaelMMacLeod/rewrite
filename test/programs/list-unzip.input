(unzip (list (pair 1 a) (pair 2 b) (pair 3 c)))
