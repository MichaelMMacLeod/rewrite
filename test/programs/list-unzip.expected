(pair (list 1 2 3) (list a b c))
